src/app/index.js

```diff
@@ -16,14 +21,21 @@ const styles = StyleSheet.create({
 // TODO: Pass in the real store to the provider
 const fakeStore = createStore(() => { });
 
-const App = () => {
-  return (
-    <View style={styles.container}>
-      <Provider store={fakeStore}>
-        <DemoButton />
-      </Provider>
-    </View >
-  );
-};
+// const App = () => {
+//   return (
+//     <View style={styles.container}>
+//       <Provider store={fakeStore}>
+//           <Event />
+//       </Provider>
+//     </View >
+//   );
+// };
+
+const App = createBottomTabNavigator({
+  Timeline: { screen: TimelineScreen },
```

Comment:
In the future, this tab navigator won't actually be the highest-level navigator, we'll probably have another one for login/splashscreen so we may want to rename this `RootNavigator` instead of `App` or something.

Also, we'll eventually be using `react-apollo` to connect the front-end code to the graphql backend, so it'll end up looking something like this:
```JS
<ApolloProvider client={client}>
    <RootNavigator />
</ApolloProvider>
```